<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SessionControlWidget</class>
 <widget class="QWidget" name="SessionControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>66</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lblObjective">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string notr="true">Objective</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="4">
    <widget class="QCheckBox" name="chkObjectiveDone">
     <property name="text">
      <string>Objectif atteint</string>
     </property>
     <property name="icon">
      <iconset resource="../ressources.qrc">
       <normaloff>:/images/thumbup-icon.png</normaloff>:/images/thumbup-icon.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Commentaire :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtComment"/>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="QToolButton" name="btnFirst">
     <property name="minimumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Première page</string>
     </property>
     <property name="icon">
      <iconset resource="../ressources.qrc">
       <normaloff>:/images/first-icon.png</normaloff>:/images/first-icon.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QToolButton" name="btnPrevious">
     <property name="minimumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Page précédente</string>
     </property>
     <property name="icon">
      <iconset resource="../ressources.qrc">
       <normaloff>:/images/leftarrow-icon.png</normaloff>:/images/leftarrow-icon.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QToolButton" name="btnNext">
     <property name="minimumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Page suivante</string>
     </property>
     <property name="icon">
      <iconset resource="../ressources.qrc">
       <normaloff>:/images/rightarrow-icon.png</normaloff>:/images/rightarrow-icon.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QToolButton" name="btnLast">
     <property name="minimumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Dernière page</string>
     </property>
     <property name="icon">
      <iconset resource="../ressources.qrc">
       <normaloff>:/images/last-icon.png</normaloff>:/images/last-icon.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>30</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="6">
    <widget class="QToolButton" name="btnFinish">
     <property name="minimumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Terminer la séance</string>
     </property>
     <property name="icon">
      <iconset resource="../ressources.qrc">
       <normaloff>:/images/checkround-icon.png</normaloff>:/images/checkround-icon.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../ressources.qrc"/>
 </resources>
 <connections/>
</ui>
